lib/toot/subscriptions_service.rb

```diff
@@ -1,9 +1,43 @@
 module Toot
   class SubscriptionsService
 
-    def call(env)
+    def self.call(env)
```

Comment:
where are we changing the usage of this?

Reply:
The usage was always via `self.call`.  Down below I've removed the old `self.call` and changed it to this.